test/GenerationSandbox.Tests/BasicTests.cs

```diff
@@ -297,12 +297,12 @@ public void GetAllWindowsInfo()
                     {
                         if (PInvoke.GetWindowText(handle, windowNameChars, bufferSize) == 0)
                         {
-                            int errorCode = Marshal.GetLastWin32Error();
-                            if (errorCode != 0)
-                            {
-                                throw new Win32Exception(errorCode);
-                            }
-
+                            // Don't throw exceptions, this can fail for many reasons on dev machines.
```

Comment:
Should this just be removed if we are commenting this out or is it still useful to run uncommented locally occasionally?